autoscale/src/main/java/com/sequenceiq/periscope/repository/ClusterRepository.java

```diff
@@ -19,16 +19,16 @@
     Cluster findByStackId(@Param("stackId") Long stackId);
 
     @Query(" SELECT c FROM Cluster c LEFT JOIN FETCH c.clusterPertain " +
-            " WHERE c.stackCrn = :stackCrn and c.clusterPertain.workspaceId = :workspaceId")
-    Optional<Cluster> findByStackCrnAndWorkspaceId(@Param("stackCrn") String stackCrn, @Param("workspaceId") Long workspaceId);
+            " WHERE c.stackCrn = :stackCrn and c.clusterPertain.tenant = :tenant")
+    Optional<Cluster> findByStackCrnAndTenant(@Param("stackCrn") String stackCrn, @Param("tenant") String tenant);
 
     @Query(" SELECT c FROM Cluster c LEFT JOIN FETCH c.clusterPertain " +
-            " WHERE c.stackName = :stackName and c.clusterPertain.workspaceId = :workspaceId")
-    Optional<Cluster> findByStackNameAndWorkspaceId(@Param("stackName") String stackName, @Param("workspaceId") Long workspaceId);
+            " WHERE c.stackName = :stackName and c.clusterPertain.tenant = :tenant")
+    Optional<Cluster> findByStackNameAndTenant(@Param("stackName") String stackName, @Param("tenant") String tenant);
 
     @Query(" SELECT c FROM Cluster c LEFT JOIN FETCH c.clusterPertain " +
```

Comment:
Where do we use this ClusterID?
I'm ok with using Name and Tenant since it might come handy on the CLI, but we are not really using clusterID anywhere.

Reply:
This is not used now since Id based endpoints is disabled in periscope now. This was used in Id based AlertEndpoint which is disabled but not deleted.